package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": "true",
+  "scripts": {
+    "build": "netlify build",
+    "postinstall": "cd website; echo 'installing website dependencies'; npm i; echo 'installing plugin dependencies'; cd plugins/pretalx-fetch; npm i"
```

Comment:
Why not leave the postinstall for the plugin dependency on website/package.json?